app/backend/views/default.jsx

```diff
@@ -45,7 +45,7 @@ class DefaultView extends Component {
                 <div id='navbar' className='collapse navbar-collapse'>
                   <ul className='nav navbar-nav'>
                     <li><a href='/about'>About</a></li>
-                    <li><a href='/pricing'>Pricing</a></li>
+                    <li className={showPricing ? 'visible' : 'hidden'}><a href='/pricing'>Pricing</a></li>
```

Comment:
@georgiana-b 

If we show pricing, then we need to show a URL to a pricing page, not in this codebase. Sorry if that was not clear from the way the issue is written.